rotkehlchen/globaldb/handler.py

```diff
@@ -344,11 +344,18 @@ def search_assets_levenshtein(
                 if entry[1] is None and entry[2] is None:
                     continue
 
-                lev_dist_name = levenshtein(substring_search, entry[1].lower()) if entry[1] is not None else 0  # noqa: E501
-                lev_dist_symbol = levenshtein(substring_search, entry[2].lower()) if entry[2] is not None else 0  # noqa: E501
+                lev_dist_name = levenshtein(substring_search, entry[1].casefold()) if entry[1] is not None else 0  # noqa: E501
+                lev_dist_symbol = levenshtein(substring_search, entry[2].casefold()) if entry[2] is not None else 0  # noqa: E501
                 lev_dist_min = min(lev_dist_name, lev_dist_symbol)
                 # the maximum levenshtein distance that should be accepted.
                 if lev_dist_min <= LEVENSHTEIN_DISTANCE_MATCH_THRESHOLD:
+                    substring_not_in_result = (
+                        (isinstance(entry[1], str) and substring_search not in entry[1].casefold()) or  # noqa:E501
```

Comment:
why the isinstance check? When can it not be a string?

Reply:
An asset has name & symbol to be nullable